src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/CreateDescriptorsValidation.feature

```diff
@@ -162,112 +157,109 @@ Feature: Descriptors are a set of mechanisms to support flexible enumerations or
                   """
                     {
                         "codeValue": "                      ",
-                        "description": "                    ",
-                        "namespace": "uri://ed-fi.org/AbsenceEventCategoryDescriptor",
+                        "namespace": "  ",
```

Comment:
The scenario says that required attributes only have spaces, but `namespace` was not "blank" before. I fixed that.